server/src/main/java/io/spine/server/entity/Transaction.java

```diff
@@ -285,21 +282,30 @@ private void commitChangedState(@NonValidated S newState) {
             commitAttributeChanges();
             EntityRecord newRecord = createRecord();
             afterCommit(entityBeforeTransaction, newRecord);
-        } catch (ValidationException exception) {  /* Could only happen if the state
+        } catch (ValidationException validationException) {  /* Could only happen if the state
                                                       has been injected not using
                                                       the builder setters. */
-            InvalidEntityStateException invalidStateException = of(exception);
```

Comment:
Please fix the comment alignment.

Reply:
The catch block no longer has the branch related to `ValidationException`. We do not throw it anymore.